pkg/plugins/manager/testdata/test-app-with-roles/MANIFEST.txt

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
question: why does the plugin need to be signed? Asking out of curiosity.

Reply:
When loading an external plugin Grafana per default requires it to be signed

Reply:
Gotcha, thanks for the explanation :)